### PR TITLE
Add Privilege (PRVG) token

### PR DESCRIPTION
**Pull request by kthomas**
Add PRVG token

Official Etherscan: https://etherscan.io/token/0x2Ec3275f7aCe4044e499823F511cd58250be8E3d

### CODE REVIEW

File: contract-map.json [hunk 1]
```diff
@@ -1,4 +1,11 @@
 {
+  "0x2Ec3275f7aCe4044e499823F511cd58250be8E3d": {
+    "name": "Privilege",
+    "logo": "prvg.png",
```
Comment (MRabenda) [inline contract-map.json:4]: SVG is required

Comment (kthomas) [reply contract-map.json]: ty 🙏🏻

**Review by Newman1515 — APPROVED** [review]
Ok

**Review by Newman1515 — CHANGES_REQUESTED** [review]
Nice

**Comment by kthomas** [comment]
What the actual fuck @MRabenda ???

Can you please merge this?

**Comment by MRabenda** [comment]
@kthomas - this SVG https://github.com/MetaMask/contract-metadata/blob/34a584f1d1ecf6b2356e9e66861ee99d1f9aeed0/images/prvg.svg has over 300kB - regular one should be ~ 10kB are you able to decrease the size?

**Comment by kthomas** [comment]
> @kthomas - this SVG https://github.com/MetaMask/contract-metadata/blob/34a584f1d1ecf6b2356e9e66861ee99d1f9aeed0/images/prvg.svg has over 300kB - regular one should be ~ 10kB are you able to decrease the size?

@MRabenda  🤦🏻‍♂️  trust but verify eh? :)

It seems to have been able to be reduced down to 15k. Not sure why that was such a pita but seems within reason now.

Thanks 🍻

**Comment by kthomas** [comment]
@MRabenda -- they got it down to 8k 🙏🏻 

I guess that SVG is just ridiculous 🤷🏻

**Review by MRabenda — APPROVED** [review]

**Comment by MRabenda** [comment]
Thanks for your contribution!

**Pull request merged**